DataScience/ProbabilisticParser/parser.py

```diff
@@ -50,12 +51,11 @@ def parse(raw_string):
     :return: a list of tokens and labels
     :rtype: list
     """
-
-    tokens = t.tokenize(raw_string)
+    tokens = tok.tokenize(raw_string)
```

Comment:
tokenize function should return empty list by default (instead of checking a boolean in the following 2 lines)